src/main/drivers/max7456.c

```diff
@@ -183,12 +188,12 @@ uint16_t maxScreenSize = VIDEO_BUFFER_CHARS_PAL;
 // we write everything in screenBuffer and set a dirty bit
 // in screenIsDirty to upgrade only changed chars this solution
 // is faster than redrawing the whole screen on each frame
-static uint8_t screenBuffer[VIDEO_BUFFER_CHARS_PAL] ALIGNED(4);
+static uint16_t screenBuffer[VIDEO_BUFFER_CHARS_PAL] ALIGNED(4);
```

Comment:
As we discussed on Slack. Probably it would be better to store attributes in a separate buffer and pack attributes to two adjacent characters into a single `uint8_t`.

Reply:
As discussed on Slack a bit later, we end up using a lot of FLASH to save those 240 bytes of BSS, so it seems not worth it right now. Plus, we had already saved ~600 bytes from BSS when moving from the double buffer to a bitarray, so we still reduced the memory used by the MAX7456 overall.